lib/pure/strutils.nim

```diff
@@ -119,6 +130,10 @@ const
     ##   doAssert "01234".find(invalid) == -1
     ##   doAssert "01A34".find(invalid) == 2
 
+  Special* = AllChars - Letters - Digits
```

Comment:
not sure we need this category, it's subjective

Reply:
Alright!